lib/index.js

```diff
@@ -221,6 +245,9 @@ export default class HyperList {
     const from = estFrom > total ? total : estFrom < 0 ? 0 : estFrom
     const estTo = from + this._cachedItemsLen
     const to = estTo > total ? total : estTo < 0 ? 0 : estTo
+    const fastCache = []
```

Comment:
I'll refactor this part soon, I'll open another PR based on your branch in case we disagree on some changes :).